sklearn/inspection/tests/test_partial_dependence.py

```diff
@@ -268,6 +312,19 @@ def test_partial_dependence_easy_target(est, power):
 
     assert r2 > .99
 
+    predictions, values = individual_conditional_expectation(
```

Comment:
Either parametrize the test or just move the test in a new file dedicated to ICE.